roles/openshift_node/tasks/upgrade/config_changes.yml

```diff
@@ -27,6 +27,13 @@
 - name: Update systemd units
   import_tasks: ../systemd_units.yml
 
+# Update the cri-o pause image to use latest version after an upgrade
+- name: Update cri-o pause image
+  lineinfile:
+    dest: /etc/crio/crio.conf
```

Comment:
I don't think `/etc/crio/crio.conf` exists on the environments that docker is the container runtime.

Reply:
Right, it wouldn't. Added a check to only do this if /etc/crio/crio.conf exists.